src/lib/FlightTasks/tasks/Auto/FlightTaskAuto.hpp

```diff
@@ -86,6 +87,8 @@ class FlightTaskAuto : public FlightTask
 	 */
 	void setYawHandler(WeatherVane *ext_yaw_handler) override {_ext_yaw_handler = ext_yaw_handler;}
 
+	CollisionPrevention _collision_prevention;	/**< collision avoidance setpoint amendment */
```

Comment:
@baumanta Should we call it prevention in the comment as well? Or might as well just get rid of the comment, the variable name pretty much says what it does.

Reply:
ah yes of course I think I copy pasted that form somewhere... I'll hunt down the other one as well ;)